setup.py

```diff
@@ -36,7 +36,7 @@
       packages=find_packages(),
       install_requires=['indra', 'boto3', 'jsonpickle', 'kappy==4.0.94',
                         'pygraphviz', 'fnvhash', 'sqlalchemy<1.4',
-                        'inflection', 'pybel==0.15',
+                        'inflection', 'pybel==0.15', 'requests',
```

Comment:
We probably don't have to install `requests` here as it's installed in INDRA (and we already used requests in several other places in EMMAA).